src/funcx_common/task_storage/default_storage.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import os
+import typing as t
+
+from .base import TaskStorage
+from .redis import ImplicitRedisStorage
+from .s3 import RedisS3Storage
+
+log = logging.getLogger(__name__)
+
+
+DEFAULT_REDIS_STORAGE_THRESHOLD: int = 20000
+
+
+def _get_redis_storage_threshold() -> int:
+    val = os.getenv("FUNCX_REDIS_STORAGE_THRESHOLD")
+    if val is not None:
+        try:
+            return int(val)
+        except ValueError:
```

Comment:
If the env var were to be mangled, I'd expect a failure to start rather than using a default.

Reply:
That's fair. Why don't I switch the `log.warning` call to a `raise ValueError(...) from e` with the same message. I only cared about being able to get a clear message somewhere about what happened. It can come in the form of an error.

Reply:
:+1:  Better to fail early and obviously when there's a misconfiguration.